test/Table_Tests/src/Common_Table_Operations/Conversion_Spec.enso

```diff
@@ -408,11 +366,11 @@ add_specs suite_builder setup =
             t = table_builder [["X", [Nothing, 1, 2, 3000]], ["Y", [Nothing, True, False, True]]]
 
             c1 = t.at "X" . cast Value_Type.Char
-            c1.value_type.is_text . should_be_true
+            c1.value_type . should_be_a (Value_Type.Char ...)
```

Comment:
why is this better than `.is_text`?

Reply:
Because if the test fails, `is_text.should_be_true` gives us "expected False to be True" which tells me nothing.

The new approach tells me e.g. "expected ... built with constructor Char but got ... built with constructor Integer" - which is **much more informative** as to what went wrong.